com.w1hkj.flcluster.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.w1hkj.flcluster
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: flcluster
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --persist=.flcluster
+  - --persist=.fltk
+  - --persist=.nbems
```

Comment:
Yeah, it really looks like that this dir is not used by flcluster. Feel free to remove it.